### PR TITLE
Removing obsolete Links

### PR DESCRIPTION
**Pull request by 1Maxnet1**
I removed some broken links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-**http://todotxt.org** [![](https://img.shields.io/gitter/room/todotxt/Lobby.svg)](https://gitter.im/todotxt/Lobby)
+**https://todotxt.org** [![](https://img.shields.io/gitter/room/todotxt/Lobby.svg)](https://gitter.im/todotxt/Lobby)
```
Comment (waldyrious) [inline README.md:1]: This change cannot be made yet — see #32.

Comment (1Maxnet1) [reply README.md:1]: When I open the https-version of the page it's already available online, but the connection ist not secure. De facto the same as http, isn't it? So we can make the change and wait for ginatrapani to update the DNS entry.

But if not I will remove this commit, so we can merge the other ones.

Comment (waldyrious) [reply README.md:1]: Don't you get a browser warning if you visit the https link? I don't think that would be an equivalent experience for the user, compared to visiting an http page and getting there without issues — even though technically the site is indeed equally unsecured in both.

Comment (1Maxnet1) [reply README.md:1]: Yes I do. Got your point. So I'll remove that.

File: index.html [hunk 2]
```diff
@@ -124,7 +124,7 @@ <h3>What Users Are Saying</h3>
 					<br/><br/>
 					</span>
 					<h3>Developers</h3>
-					<p>Todo.txt CLI and Todo.txt Touch are proudly open source. Browse the source code for the <a href="https://github.com/todotxt/todo.txt-cli">CLI</a>, <a href="https://github.com/todotxt/todo.txt-touch-ios">iOS</a>, and <a href="https://github.com/todotxt/todo.txt-touch">Android</a>.</p>
```
Comment (karbassi) [inline index.html:127]: @1Maxnet1 Why was the CLI link removed?

**Comment by karbassi** [comment]
Thanks for the changes/PR @1Maxnet1 
Once you bring back the CLI link, I'll merge.

File: index.html (hunk 2 above)
Comment (1Maxnet1) [reply index.html:127]: To be honest, i don't know, why i should have done this. Mistake of mine. I updated the other links in the line to work again and brought back CLI-Link. Hopefully it's ready to get merged now. Sorry for the chaotic PR.

**Comment by karbassi** [comment]
Thanks @1Maxnet1 for the PR. 

Merged.

**Pull request merged**